app/src/main/java/com/keylesspalace/tusky/adapter/FollowRequestViewHolder.kt

```diff
@@ -87,9 +88,9 @@ class FollowRequestViewHolder(
         binding.notificationTextView.visible(showHeader)
         val formattedUsername = itemView.context.getString(R.string.post_username_format, account.username)
         binding.usernameTextView.text = formattedUsername
-        val avatarRadius = binding.avatar.context.resources.getDimensionPixelSize(
-            R.dimen.avatar_radius_48dp
-        )
+        binding.accountNote.text =
+            account.note.parseAsMastodonHtml().emojify(account.emojis, binding.accountNote, animateEmojis)
```

Comment:
There are accounts without bio, lets hide the view in that case?
![Screenshot_20230424_173104](https://user-images.githubusercontent.com/10157047/234044511-3d248d83-693e-4e48-8ef5-fd7b55f6338c.png)